builtin/providers/aws/resource_aws_dms_endpoint.go

```diff
@@ -0,0 +1,334 @@
+package aws
+
+import (
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/private/waiter"
+	dms "github.com/aws/aws-sdk-go/service/databasemigrationservice"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceAwsDmsEndpoint() *schema.Resource {
```

Comment:
The endpoint create and update are eventually consistent. I didn't put a wait on these because the update is very fast and it is hard to properly tell difference between the resource going from status of available to modifying to available vs. never leaving available.

Any suggestions on the proper handling of this?